functest/package_test.go

```diff
@@ -11,23 +11,35 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package functest
 
 import (
+	"bytes"
 	"path/filepath"
 	"testing"
 
+	cmd "github.com/appsody/appsody/cmd"
 	"github.com/appsody/appsody/cmd/cmdtest"
 )
 
 func TestPackage(t *testing.T) {
 	sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
 	defer cleanup()
 
-	args := []string{"stack", "package", filepath.Join("..", "cmd", "testdata", "starter")}
-	_, err := cmdtest.RunAppsody(sandbox, args...)
+	var outBuffer bytes.Buffer
+	log := &cmd.LoggingConfig{}
+	log.InitLogging(&outBuffer, &outBuffer)
+
+	stackDir := filepath.Join("..", "cmd", "testdata", "starter")
+	err := cmd.CopyAllContents(log, stackDir, sandbox.ProjectDir)
+	if err != nil {
+		t.Errorf("Problem copying %s to %s: %v", stackDir, sandbox.ProjectDir, err)
+	} else {
+		t.Logf("Copied %s to %s", stackDir, sandbox.ProjectDir)
+	}
 
+	args := []string{"stack", "package"}
+	_, err = cmdtest.RunAppsody(sandbox, args...)
 	if err != nil {
```

Comment:
There should be more validation that the `stack package` command works here. Making sure there was no error is good, but we should also check for instance that a new image exists with the proper tag. And a proper template tar exists.

This comment should be resolved in a separate PR. The current PR is about updating the sandboxing not improving the test cases, but I wanted to point it out while I was here reviewing.